src/frontends/tensorflow_lite/src/input_model.cpp

```diff
@@ -85,8 +114,10 @@ void InputModel::InputModelTFLiteImpl::load_model() {
     for (; !m_graph_iterator->is_end(); m_graph_iterator->next()) {
         const auto& decoder = m_graph_iterator->get_decoder();
 
-        if (auto tensor_decoder = std::dynamic_pointer_cast<DecoderFlatBufferTensors>(decoder)) {
-            auto tensor_place = tensor_decoder->decode_tensor(m_input_model);
+        if (auto tensor_decoder = std::dynamic_pointer_cast<DecoderBaseTensor>(decoder)) {
+            auto tensor_place = decode_tensor(tensor_decoder->get_tensor_info(), m_input_model);
+            tensor_place->set_input_index(tensor_decoder->get_input_idx());
```

Comment:
Why can't we do it inside the decode_tensor by a single call?

Reply:
no, because `decode_tensor` handles both `DecoderBaseOperation` and `DecoderBaseTensor`. Only `DecoderBaseTensor` has `get_input_idx()` method